### PR TITLE
Bom Verification Front End 

### PR DESCRIPTION
**Pull request by RossStewart**

### CODE REVIEW

File: src/Service.Host/client/src/components/BomVerificationHistoryUtility/BomVerificationHistory.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                        message="Create Successful"
+                        timeOut={3000}
+                    />
+                    <Grid item xs={12}>
+                        <Typography variant="h3">BOM Verification Entry</Typography>
+                    </Grid>
+                    <Grid item xs={12}>
+                        <InputField
+                            fullWidth
+                            value={`${item?.partNumber}`}
```
Comment (lewisrenfrew) [inline src/Service.Host/client/src/components/BomVerificationHistoryUtility/BomVerificationHistory.js:58]: you don't need the [`  template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here (and same for most of the other 'value' props for fields on these forms)

File: src/Service.Host/client/src/components/BomVerificationHistoryUtility/BomVerificationHistory.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                            value={`${item?.dateVerified}`}
+                            label="Date Verified"
+                            disabled
+                        />
+                    </Grid>
+                    <Grid item xs={12}>
+                        <InputField
+                            fullWidth
+                            value={`(${item?.verifiedBy})`}
+                            label="Verified By (UC)"
```
Comment (lewisrenfrew) [inline src/Service.Host/client/src/components/BomVerificationHistoryUtility/BomVerificationHistory.js:78]: Drop the (UC) from this label - user codes don't exist in the brave new world

File: src/Service.Host/client/src/components/BomVerificationHistoryUtility/CreateBomVerificationHistory.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                        value={`${item.dateVerified}`}
+                        label="Date Verified"
+                        disabled
+                    />
+                </Grid>
+                <Grid item xs={12}>
+                    <InputField
+                        fullWidth
+                        value={`${currentUserName} (${item.verifiedBy})`}
+                        label="Verified By (UC)"
```
Comment (lewisrenfrew) [inline src/Service.Host/client/src/components/BomVerificationHistoryUtility/CreateBomVerificationHistory.js:109]: delete UC again

File: src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    beforeEach(() => {
+        cleanup();
+        jest.clearAllMocks();
+        useSelector.mockImplementation(callback => callback(stateWithItem));
+        render(<BomVerificationHistory />);
+    });
+
+    test('Should render details', () => {
+        expect(screen.getByDisplayValue('SK HUB')).toBeInTheDocument();
+        expect(screen.getByDisplayValue('2023-01-26T13:55:20.0000000')).toBeInTheDocument();
```
Comment (lewisrenfrew) [inline src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js:92]: does this imply that youre rendering a full isoString date on the form? if so you probably want a more user friendly way of displaying the date

File: src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    });
+    test('Should add with values on create...', async () => {
+        const searchInput = screen.getByLabelText('Part Number');
+        fireEvent.change(searchInput, { target: { value: 'TON IC' } });
+        fireEvent.keyDown(searchInput, { key: 'Enter', code: 'Enter', keyCode: 13 });
+        const result = screen.getByText('TON IC');
+        fireEvent.click(result);
+
+        // need to wait for modal to close
+        const input = await screen.getByLabelText('Remarks', undefined, { timeout: 5000 });
```
Comment (lewisrenfrew) [inline src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js]: you can delete this 'await' keyword and the additional parameters passed to getByLabelText - don't think they're doing anything here

**Review by lewisrenfrew — APPROVED** [review]

File: src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js (hunk 5 above)
Comment (RossStewart) [reply src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js]: Sorted these in a small update during stand up!

File: src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js (hunk 4 above)
Comment (RossStewart) [reply src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js:92]: Yeah, good point.

File: src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+import CreateBomVerificationHistory from '../BomVerificationHistoryUtility/CreateBomVerificationHistory';
+import BomVerificationHistory from '../BomVerificationHistoryUtility/BomVerificationHistory';
+import bomVerificationHistoryActions from '../../actions/bomVerificationHistoryActions';
+
+jest.mock('react-redux', () => ({
+    ...jest.requireActual('react-redux'),
+    useSelector: jest.fn()
+}));
+
+jest.mock('react-router', () => ({
```
Comment (lewisrenfrew) [inline src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js:19]: you might not need this jest.mock?

File: src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js (hunk 5 above)
Comment (lewisrenfrew) [reply src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js]: should delete the comment too

File: src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js (hunk 6 above)
Comment (RossStewart) [reply src/Service.Host/client/src/components/__tests__/BomVerificationHistory.test.js]: Good spot! Removed.

**Pull request merged**